authorization/redhat-ai-dev-users.yaml

```diff
@@ -0,0 +1,21 @@
+# This group provides cluster-wide edit access on the redhat-ai-dev team OpenShift cluster.
+# Your user *must* already exist on the cluster before adding it to this list
+kind: Group
+apiVersion: user.openshift.io/v1
+metadata:
+  name: redhat-ai-dev-users
+users:
+  - yangcao77
+  - michael-valdron
+  - maysunfaisal
+  - elsony
+  - thepetk
```

Comment:
Out of curiosity: Why we add both the github handle and the RH handle?

Reply:
Oops, didn't see I left your GitHub user name in there. Since you have cluster-admin permissions, you don't need to be in this file.

We used to use GitHub to log in to the cluster, but we've now switched to Kerberos auth via internal SSO. I'll clean this up to remove any other GitHub usernames